examples/tts/conf/audio_codec/encodec_24000.yaml

```diff
@@ -162,9 +162,16 @@ exp_manager:
   exp_dir: null
   name: ${name}
   create_tensorboard_logger: true
-  create_checkpoint_callback: true
-  create_wandb_logger: false
+  create_wandb_logger: true
+  wandb_logger_kwargs:
+    name: ${name}
+    project: ???
```

Comment:
Should we keep the convention that all `???` be at the top of the file? I think TTS is the only domain which does it currently.

Reply:
I think for wandb the "name" field is supposed to be the name of a specific run to track. "project" is the category to sort runs by.

So project could be set to ${name} or some other hardcoded value (which user can override). Name should be a unique value the user provides for each run (or if not provided it generates a random name like _happy-giraffe-42_)